test/parallel/test-crypto.js

```diff
@@ -54,19 +54,18 @@ assert.throws(function() {
 
 
 function assertSorted(list) {
-  // Array#sort() modifies the list in place so make a copy.
-  const sorted = list.slice().sort();
-  assert.deepStrictEqual(list, sorted);
+  // list is sorted and contains no duplicates
+  assert(list.every(
+    (val, index, array) => index === array.length - 1 || val < array[index + 1]
```

Comment:
While it works as commented, this is a pretty obfuscated way of doing both a sorted test and a uniqueness test in a single statement. I prefer the approach in https://github.com/nodejs/node/blob/20efbffaf8c5f4af94da31c95ba90e77a2e00e08/test/parallel/test-process-getgroups.js#L30-L32, though in hind-sight, maybe that `unique()` function should not *also* be sorting. Maybe `unique()` should be in `test/common.js`?